packages/block-library/src/tag-cloud/index.php

```diff
@@ -15,9 +15,12 @@
 function render_block_core_tag_cloud( $attributes ) {
 	$args      = array(
 		'echo'       => false,
+		'unit'       => preg_replace( '/\d+/', '', $attributes['smallestFontSize'] ),
```

Comment:
All these might need escaping for sanitisation. Also check if in our code base some util exists or similar implementation for extracting the value/unit. It's a common use case and there might probably be somewhere.

Reply:
Indeed sanitisation is required. I checked function in code reference but no luck. I asked in the `#core` channel and got [reply](https://wordpress.slack.com/archives/C02RQBWTW/p1645816392566379). I don't think there is any function available. Also, for sanitisation is it okay to use `sanitize_text_field` method?

Reply:
--cc @getdave that did a lot of escaping recently 😄 

Also noting that we could have decimals, so the regex should take that into account.

Reply:
I don't think we need to escape the args to this core function. I'd be concerned if we were creating an arbitary HTML string but this one is generated by a Core function and the one thing I learnt from all the escaping I did, it that we should trust Core functions, especially those which have [available filters](https://github.com/WordPress/wordpress-develop/blob/8cc9e53847f8a46258d9e42c29f8d524ec0af592/src/wp-includes/category-template.php#L765).

Reply:
I have updated the regex to handle decimal cases.